### DISCUSSION TITLE
Can PCM moniter memory bandwidth caused by DMA?

### DISCUSSION THREAD

**Discussion by Jiangrijin** [opcm/pcm#302] [Q&A] [answered]
I am looking for a tool to moniter VM's memory bandwidth. I wonder if the output of PCM includes DMA's bandwidth like network bandwidth. 
And as far as I know, PCM counts memory bandwidth generated by each socket, so I also want to know can PCM count memory bandwidth generated by each core.

**Comment by opcm** [comment, marked as answer]
The output from READ and WRITE columns (per-socket) includes DMA memory traffic. The LMB an RMB pcm metrics are per-core (but obviously do not contain DMA).
